app/src/main/java/ch/epfl/culturequest/social/Profile.java

```diff
@@ -22,14 +18,10 @@ public class Profile extends Observable {
 
     private String uid, name, username, email, phoneNumber;
     private String profilePicture;
-    private List<Post> images;
+    private ArrayList<Post> posts;
```

Comment:
Better to use an interface for attributes as the implementation matters less. But more importantly, I think that we can remove the list of posts from the user so that it is not put in the database as is. We shouldn't fetch all the posts when we get a profile, this is a separate collection.

Reply:
Yes you are right, I did that in order to help the person who read the class attribute now that we always use it as an ArrayList but at the next meeting we need to decide a convention and refactor everything. Because the declaration of the lists in Post and Profile is quite a mess ...

Reply:
Maybe the get posts will then fetch the database, so that accessing this method is easy for this type of query.

Reply:
Yes I get it, as it is not related to this PR let's do it on yours about the post then.